backend/fbdev.c

```diff
@@ -85,6 +97,7 @@ static bool twin_fbdev_work(void *closure)
 
     if (twin_screen_damaged(screen))
         twin_screen_update(screen);
+    twin_fbdev_cache_sync(closure);
```

Comment:
Could you explain why `twin_fbdev_cache_sync` should be called here?

Reply:
After some search, this is driver-specificed code and should not be used, I will remove this code.
This code has a side-effect on my machine that it updates the smooth display but on VirtualBox is not.